src/components/GameItem.js

```diff
@@ -4,26 +4,50 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
+
   static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  constructor() {
+    super();
+    this.state = {
+      spotted: false,
+    };
+  }
+
   onItemClicked = () => {
     // Fill this in for Wave 2!
-  }
+    this.setState({ spotted: !this.state.spotted });
     
+    if ((this.props.type === "litter") && (this.state.spotted === false)){
+      this.props.score()
```

Comment:
Make sure that you are highlighting what the purpose of that function is with its name. If it’s a callback, use the term callback, if it’s an event handler, tell us what kind of event it is. `score` is a callback.